lib/helper/REST.js

```diff
@@ -1,28 +1,45 @@
 'user strict';
 const Helper = require('../helper');
 const unirest = require('unirest');
-/**
+/*
  * REST helper to execute RESTful API calls, this is currently under development phase.
  * Usage:
  *      let response = yield I.sendGet('https://randomuser.me/api/')
  *      I.say(`I received: ${response.raw_body}`)
  * Configuration:
  *      Helper should be configured like:
  *      "helpers": {
- *          "REST": {}
+ *          "REST": {
+ *              "timeout": <numeric value>
+ *              You can either send a default timeout or not according to your needs
```

Comment:
Put something like:
 "timeout": <integer, numeric value> - You can either send a default timeout or not according to your needs.